File: Android/databaseGames/java/com/korobeynikov/databasegames/DBGames.java
package com.korobeynikov.databasegames;

import android.content.Context;
import android.database.sqlite.SQLiteDatabase;
import android.database.sqlite.SQLiteOpenHelper;
import android.widget.Toast;

import androidx.appcompat.app.AppCompatActivity;

public class DBGames extends SQLiteOpenHelper {

    MainActivity mainActivity;

    public DBGames(Context context){
        super(context,"dbGames",null,1);
        mainActivity=(MainActivity)context;
    }

    @Override
    public void onCreate(SQLiteDatabase db) {
        db.execSQL("create table Games (id integer primary key autoincrement, name text, rating integer, year integer);");
        Toast.makeText(mainActivity,"База данных успешно создана!",Toast.LENGTH_LONG).show();
    }

    @Override
    public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {

    }
}

File: Kotlin/MyGames/java/ru/korobeynikov/databasegames/DBGames.kt
package ru.korobeynikov.databasegames

import android.content.Context
import android.database.sqlite.SQLiteDatabase
import android.database.sqlite.SQLiteOpenHelper

class DBGames(context: Context) : SQLiteOpenHelper(context, "dbGames", null, 1) {

    override fun onCreate(db: SQLiteDatabase?) {
        db?.execSQL("create table Games (id integer primary key autoincrement, name text, " +
                "rating integer, year integer, genre integer);")
    }

    override fun onUpgrade(p0: SQLiteDatabase?, p1: Int, p2: Int) {}
}